woocommerce/woocommerce-ios
# Simplified Login: Update prologue screen image

Closes: #8027


### Description
As part of the simplified login iteration project this PR updates the prologue screen image for control and treatment variations. 

Design - ERGtCFLyiC9CFpHsw1XIid-fi-667%3A15640

### Testing instructions

- Reinstall and launch the app and notice the new image from Figma in the prologue screen. (design above)

### Screenshots
|Dark|Light|
|-|-|
|![Simulator Screen Shot - iPhone 12 Pro - 2022-11-04 at 14 11 24](https://user-images.githubusercontent.com/524475/199930484-6e2386e5-61df-4c25-a0dd-da9a128b1022.png)|![Simulator Screen Shot - iPhone 12 Pro - 2022-11-04 at 14 11 30](https://user-images.githubusercontent.com/524475/199930498-490a55a1-2a10-4051-97d5-901d776ac1bc.png)|



---
